README.md

```diff
@@ -1,2 +1,22 @@
 # work-experience
 Test repository for new starters to fork and make changes
+24/08/2021
+
+Anil
+
+I am a student at Valley Park school studying Computer science, IT and History. I like to play sports
+like basketball and frequnetly go to the gym. I am very hardworking and studious. I would like learn
+more about computer science in a work setting and gain invalueable experiance for a future in the
+field. 
+ ```
+{
+  "Name": "Anil",
+  "lastName": "McGill",
+  "Age": 17
+}
+```
+print("hello World!, my name is",(Name),"and I am",(Age),"years old.")
```

Comment:
`name = "Anil"
 age = "17"
 message "Hello World! My name is "+ name + " and I am " + age + " years old"
 print(message)`